lyricsbot/domains/lyricsondemand/utils.py

```diff
@@ -0,0 +1,35 @@
+"""
+Helper for `lyricsondemand` implementation.
+"""
+from string import punctuation
+
+from lyricsbot.lib import join, split
+
+
+def make_suitable_url_parameters(url_parameter):
+    """
+    Data-in is a string with spaces or without any punctuation symbol.
```

Comment:
It is overall explanation, that need to be placed below of doc-string.
First phrase of doc-string should follow imperative mood and describe what function does.

Reply:
Done.

Reply:
Done.